advanced/button-interrupt/solution/src/main.rs

```diff
@@ -12,30 +12,15 @@ use esp_idf_sys::{
 
 static mut EVENT_QUEUE: Option<QueueHandle_t> = None;
```

Comment:
`static mut` sad face. Worth a comment to say "This is unsafe, but we are careful not to enable our GPIO interrupt handler until after this value has been initialised, and then never modify it again". Or use whatever the RISC-V/ESP32-C3 equivalent of a `cortex-m::Mutex` is. https://github.com/rp-rs/rp-hal/blob/main/rp2040-hal/examples/gpio_irq_example.rs#L81 is how I solved 'share some things with an IRQ handler' for the RP2040.